hbase-common/src/test/java/org/apache/hadoop/hbase/io/crypto/TestEncryption.java

```diff
@@ -1,22 +1,24 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements. See the NOTICE file distributed with this
- * work for additional information regarding copyright ownership. The ASF
- * licenses this file to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+/**
```

Comment:
this is a file header and not a javadoc, shouldn't it be using `/*` and not `/**`?

Reply:
Fixed